examples/scenes/src/simple_text.rs

```diff
@@ -92,6 +92,7 @@ impl SimpleText {
             .glyph_transform(glyph_transform)
             .normalized_coords(var_loc.coords())
             .brush(brush)
+            .hint(false)
```

Comment:
I would expect us to want to test this?
Is it because SimpleText is also used for the headless example?

I see in Zulip that you've mentioned that  text should be on a pixel baseline, which we don't enforce in our current examples.

I still would imagine that hinting could be productively used in e.g. the stats displays?

Reply:
This is disabled for the test scenes because we have some animated text which doesn’t work well with hinting (it snaps to pixels as it animates causing annoying jitter) and everything currently uses the same code path. We can definitely change this but I wanted to get the functionality in for xilem where hinting is more important.